std_execution.bs

```diff
@@ -553,6 +553,446 @@ merged into a single one in an asynchronous fashion.
+compiled out and having little to no run-time overhead.
+
+If the sender doesn't call `execution::get_stop_token()`, for example because the operation does not support
+cancellation, then it will simply not respond to stop-requests from the caller.
+
+Note that stop-requests are generally racy in nature as there is often a race betwen an operation completing
+naturally and the stop-request being made. If the operation has already completed or past the point at which
+it can be cancelled when the stop-request is sent then the stop-request may just be ignored. Thus an application
+will typically need to be able to cope with senders that might ignore a stop-request anyway.
+
+### Cancellation and eager operations ###
+
+Some of the algorithms proposed in this paper support executing their logic eagerly. i.e. before the
+returned sender has been connected to a receiver and started. This has a number of implications with
+respect to cancellation.
+
+One of the questions that arises with APIs that can potentially return eagerly-executing senders is
+"What happens when those senders are destructed without a call to `execution::connect`?" or similarly,
+"What happens if a call to `execution::connect` is made, but the returned operation-state is destroyed
+before `execution::start` is called on that operation-state"?
+
+In these cases, the operation represented by the sender is potentially executing concurrently in another
+thread at the time that the destructor of the sender and/or operation-state is running. In the case that
+the operation has not completed executing by the time that the destructor is run we need to decide what
+the semantics of the destructor is.
+
+There are three main strategies that can be adopted here:
+1. Make this undefined-behaviour - the caller must ensure that any eagerly-executing sender is always joined by connecting and starting that sender.
+   This approach is generally pretty hostile to programmers, particularly in the presence of exceptions, and is thus not recommended.
+2. Detach from the computation - let the operation continue in the background - like an implicit call to `std::thread::detach()`.
+   While this approach can work in some circumstances for some kinds of applications, in general it is also pretty user-hostile
+   as it makes it difficult to reason about the safe destruction of resources used by these eager operations. In general, you'll
+   need some kind of garbage collection, e.g. `std::shared_ptr`, to ensure resources are kept alive until the operations complete.
+3. Block in the destructor until the operation completes.
+   This approach is probably the safest to use as it preserves the structured nature of the concurrent operations,
+   but also introduces the potential for deadlocking the application if the completion of the operation depends on
+   the current thread making forward progress.
```

Comment:
Mentioned the history of `jthread` maybe? It seems to be justified the same way. And "we went on that path before" is a good argument imo